src/components/TitleBody/TitleBody.scss

```diff
@@ -0,0 +1,15 @@
+.body {
+  margin-bottom: 4.5rem;
+
+  li + li {
+    padding-top: 12px;
+  } 
```

Comment:
what is this styling?

Reply:
was following the design 
![image](https://user-images.githubusercontent.com/29209916/106820126-ae60af00-662f-11eb-8a38-a4485d13a07e.png)